### PR TITLE
adding OP_CHECKLOCKTIMEVERIFY

### PR DESCRIPTION
**Pull request by PierreRochard**
Addresses https://github.com/petertodd/python-bitcoinlib/issues/34

### CODE REVIEW

**Comment by PierreRochard** [comment]
Just realized OP_CHECKLOCKTIMEVERIFY is not "official" yet - I'll leave this open for comments

**Comment by petertodd** [comment]
Yeah, Bitcoin Core hasn't changed OP_NOP2 to OP_CHECKLOCKTIMEVERIFY yet; inclined to merge this or something like it after the next release of python-bitcoinlib.

File: bitcoin/core/script.py [hunk 1]
```diff
@@ -1061,4 +1065,4 @@ def SignatureHash(script, txTo, inIdx, hashtype):
         'RawSignatureHash',
         'SignatureHash',
         'IsLowDERSignature',
-)
+)
```
Comment (epheph) [inline bitcoin/core/script.py]: Super pedantic, but can you revert the deletion of the newline?

Comment (PierreRochard) [inline bitcoin/core/script.py]: @epheph absolutely, I didn't mean to do that. I'm also not very experienced with git and interested in learning, what would be the best way to revert?

**Comment by epheph** [comment]
It isn't really a "revert" in the git sense, since if you did a "git revert" at the head of your branch, it would undo all changes in a brand new commit and you'd have to re-add all the changes you wanted back. I would just do a simple new git commit with the eol added. Curiously, what editor did this?

**Pull request merged**